etl/steps/data/garden/animal_welfare/2023-08-14/number_of_farmed_fish.meta.yml

```diff
@@ -0,0 +1,36 @@
+definitions:
+  common: |
+    This study estimates the annual number of farmed fish slaughtered for food using data on aquaculture production tonnages (from the Food and Agriculture Organization of the United Nations) and mean weights estimated from available fish size data (from a range of internet sources).
+
+    Note that this estimate does not include:
+    - Non fish species, e.g. lobsters.
```

Comment:
`Non-fish`?

Reply:
Also, I don't see this filtered out in the dataset. (column `Fish species?`)

Reply:
If you select `Fish species? = N` you'll see that none of those rows have an estimate on mean weights or numbers. I'm now adding data for decapod crustaceans (in a separate PR).